Godeps/_workspace/src/k8s.io/kubernetes/pkg/api/types.go

```diff
@@ -2507,6 +2505,30 @@ type SecurityContextConstraints struct {
 	Groups []string
 }
 
+// FS Type gives strong typing to different file systems that are used by volumes.
+type FSType string
```

Comment:
need a reflective test to detect drift if a new field is added to `VolumeSource`